### PR TITLE
First test of GitHub actions

### LINKED ISSUE #107: Move from Travis to github actions
Move from Travis to github actions: we are experiencing long build delay using Travis, and according to changes in Travis management/strategy it may be safer to start looking at alternatives like GitHub actions.
https://docs.github.com/en/free-pro-team@latest/actions/quickstart

https://developer.okta.com/blog/2020/05/18/travis-ci-to-github-actions

https://github.com/marketplace/actions/hugo-setup

To be done:
- [ ] Mardkown linting
- [ ] Building site with hugo
- [ ] deploy on PR merge

### PR DESCRIPTION
**Pull request by gwarf**
First test of GitHub actions.
Fixes #107 

To be merged once all feature of travis will be supported.

To be done:
- [ ] Mardkown linting
- [ ] Hugo building
- [ ] deploy on PR merge

### CODE REVIEW

**Comment by enolfc** [comment]
linter is having fun :)

File: content/en/providers/cloud-compute/configuration/_index.md [hunk 1]
```diff
@@ -13,81 +13,80 @@ infrastructure
...
-    ```
-
-1.  Update the mapping in your Keystone IdP: :
-
-    ``` {.console}
-    openstack mapping set --rules mapping.json egi-mapping
-    ```
+Follow these steps if you are using OpenID Connect to integrate with EGI:
+
+1. Create a group where users belongig to the VO will be mapped to: :
```
Comment (enolfc) [inline content/en/providers/cloud-compute/configuration/_index.md:24]:
```suggestion
1. Create a group where users belongig to the VO will be mapped to:
```

File: content/en/providers/cloud-compute/configuration/_index.md [hunk 2]
```diff
@@ -13,81 +13,80 @@ infrastructure
...
+
+   ```sh
+   group_id=$(openstack group create -f value -c id <new_group>)
+   ```
+
+1. Add that group to the desired local project: :
```
Comment (enolfc) [inline content/en/providers/cloud-compute/configuration/_index.md:30]:
```suggestion
1. Add that group to the desired local project:
```

File: content/en/providers/cloud-compute/configuration/_index.md [hunk 3]
```diff
@@ -13,81 +13,80 @@ infrastructure
...
+
+   ```sh
+   openstack role add member --group $group_id --project <your project>
+   ```
+
+1. Expand your mapping.json with the VO membership to the created group
+   (substitute `group_id` and `vo_name` as appropriate): :
```
Comment (enolfc) [inline content/en/providers/cloud-compute/configuration/_index.md:37]:
```suggestion
   (substitute `group_id` and `vo_name` as appropriate):
```

File: content/en/providers/cloud-compute/configuration/_index.md [hunk 4]
```diff
@@ -13,81 +13,80 @@ infrastructure
...
+                   "any_one_of": [
+                       "^urn:mace:egi.eu:group:<vo_name>:role=vm_operator#aai.egi.eu$"
+                   ]
+               }
+           ]
+       }
+   ]
+   ```
+
+1. Update the mapping in your Keystone IdP: :
```
Comment (enolfc) [inline content/en/providers/cloud-compute/configuration/_index.md:75]:
```suggestion
1. Update the mapping in your Keystone IdP:
```

**Review by enolfc — COMMENTED** [review]
Lots of changes to the markdown. I didn't review all of them, but I spotted some dangling `:`. 

Will you also get into hugo building here or that's another PR?

**Comment by gwarf** [comment]
You can see results at https://github.com/gwarf/documentation/actions
I'm a bit afraid by all those changes in fact :/ But it already took me quite some time...

**Comment by gwarf** [comment]
Maybe pushing linted code would be already a good first step. I'm afraid other may start to change some stuff in the meantime...

**Comment by gwarf** [comment]
markdownlint is quite more strict than mdl, it's a good thing, but it's also a pain :/

**Comment by gwarf** [comment]
Deprecated by #125 that only enable some tests so that we can fix files gradually.

**Pull request closed without merging**